src/js/App/RootApp.js

```diff
@@ -40,6 +40,7 @@ const RootApp = ({
                         </div>
                     </section>
                 </main>
+                <main className="pf-c-page__main pf-l-page__main" id="no-access"></main>
```

Comment:
We don't need to use the legacy classnames

Reply:
Legacy classnames as in `pf-l-page__main`?

Reply:
Yep!